src/util/GithubRepositoryUtil.js

```diff
@@ -0,0 +1,64 @@
+const REPO_LIST_KEY = 'repoList';
```

Comment:
It's not for public access. It's so that we don't accidentally collide one day with GitHub. For example, what if all of a sudden GitHub.io pages start storing `repositories` as a top level key in their pages? If you look at the existing localStorage, they do indeed inject localStorage data into **our** page.